flight_phase_files/HALO_RF11_20200209_info.yaml

```
@@ -22,68 +26,127 @@ remarks:
 - At the 1st Ron Brown flyover a dropsonde was launched.
 - Mostly shallow sugar clouds during the flight.
 - Two cold pools entered the circle during the 2nd circle set.
+- At the beginning of the leg inbound from NTAS, a diversion due to traffic was needed.
 segments:
-- kind: circle
-  name: c1
-  start: 2020-02-09 09:36:08
-  end: 2020-02-09 10:27:24
-- kind: circle_break
-  name: cb1to2
-  start: 2020-02-09 10:27:24
-  end: 2020-02-09 10:50:11
-- kind: circle
-  name: c2
-  start: 2020-02-09 10:50:11
-  end: 2020-02-09 11:42:45
-- kind: circle_break
-  name: cb2to3
-  start: 2020-02-09 11:42:45
-  end: 2020-02-09 11:59:21
-- kind: circle
-  name: c3
-  start: 2020-02-09 11:59:21
-  end: 2020-02-09 12:54:05
-- kind: straight_leg
-  name: leg1_outbound_NTAS
-  start: 2020-02-09 12:59:59
-  end: 2020-02-09 13:17:03
-- kind: radar_calibration_wiggle
-  name: radar_cal_wiggle1
-  start: 2020-02-09 13:17:03
-  end: 2020-02-09 13:19:42
-- kind: radar_calibration_tilted
-  name: radar_cal_tilted
+- kinds: [circle]
+  name: circle 1
+  start: 2020-02-09 09:35:28
+  end: 2020-02-09 10:32:48
+  segment_id: HALO-0209_c1
+  irregularities: [begin is 41s before first launch due to roll angle at start]
+  good_dropsondes: 11
+- kinds: [circle_break]
+  name: circle break 1/2
+  start: 2020-02-09 10:32:24
+  end: 2020-02-09 10:49:12
+  segment_id: HALO-0209_cb1
+  irregularities: []
+- kinds: [circle]
+  name: circle 2 
+  start: 2020-02-09 10:49:12
+  end: 2020-02-09 11:46:44
+  segment_id: HALO-0209_c2
+  irregularities: []
+  good_dropsondes: 10
+- kinds: [circle_break]
+  name: circle break 2/3
+  start: 2020-02-09 11:46:44
```

If you change the end of cricle 2, adjust the circle_break start.
```suggestion
  start: 2020-02-09 11:46:45
```

---

As circle 2 did not change, this is still fine.